modules/gumgumBidAdapter.js

```diff
@@ -177,7 +177,7 @@ function buildRequests (validBidRequests, bidderRequest) {
       data.fp = params.bidfloor;
     }
     if (params.inScreen) {
-      data.t = params.inScreen;
+      (utils.isNumber(params.inScreen)) ? data.pubId = params.inScreen : data.t = params.inScreen;
```

Comment:
Sorry, but as I noted in the docs PR, I don't like that you're looking at the type of the value and deciding what to do with it. This will break your Prebid Server adapter.

Reply:
@bretg - I've responded to the other PR but I'll put a brief descrip here:
tracking ids must be entered as strings while publisher ids must be entered as integers. it is shown in our documentation and onboarding guides
cc @derberbaby @WayneYang1

Reply:
Sorry, I can't accept this solution. Prebid Server requires that parameters be either strings or integers, but not both.

I'm thinking it should be easy -- just have your users enter them as strings. They will be passed as strings to Prebid Server. Your adapter can make the determination how to treat it. e.g. Number.isInteger()

If for some reason that doesn't work, then you need to add another parameter with a specific type.

Reply:
If I'm not mistaken, wouldn't passing a string into Number.isInteger consistently give us `false`?
I can regroup with the team, but adding in a new parameter has been met with a good amount of pushback and would prefer to avoid it if possible

Reply:
Ok, how about isNaN then? https://stackoverflow.com/questions/175739/built-in-way-in-javascript-to-check-if-a-string-is-a-valid-number